spec/services/manifest_builder_v3/by_resource_type/manifest_builder_v3_audio_resource_spec.rb

```diff
@@ -208,7 +209,7 @@
       user = User.first
       stub_findingaid(pulfa_id: "C0652")
       stub_findingaid(pulfa_id: "C0652_c0377")
-      IngestArchivalMediaBagJob.perform_now(collection_component: "C0652", bag_path: bag_path, user: user, member_of_collection_ids: [collection.id.to_s])
+      IngestArchivalMediaBagJob.perform_now(collection_component: "C0652", bag_path: bag_path, user: user, member_of_collection_ids: [collection.id.to_s], visibility: "open")
```

Comment:
Why was the visibility change necessary?

Reply:
Not necessary. Fixed.